### PR TITLE
Update news in preparation for v5.0.0rc6.

### PR DESCRIPTION
**Pull request by awlauria**
Signed-off-by: Austen Lauria <awlauria@us.ibm.com>

### CODE REVIEW

**Review by janjust — APPROVED** [review]

File: docs/news/news-v5.0.x.rst [hunk 1]
```diff
@@ -40,24 +40,27 @@ Open MPI version 5.0.0rc5
       libraries, rather than linked into the Open MPI core libraries.
 
 
-Changes since rc4:
+Changes since rc5:
 
-  - Various changes and cleanup to fix, and better support the static building of Open MPI.
   - The PRRTE submodule pointer has been updated to bring in the following fixes:
+    - Fixed a bug where the deprecated option ``--oversubscribe`` for ``mpirun``
```
Comment (jsquyres) [inline docs/news/news-v5.0.x.rst:46]: You need a blank line before this one

File: docs/news/news-v5.0.x.rst [hunk 2]
```diff
@@ -40,24 +40,27 @@ Open MPI version 5.0.0rc5
...
+      was not translated correctly to its new equivalent (``--map-by :oversubscribe``).
+    - Fixed a case where ``--map-by ppr:x:oversubscribe`` would not work correctly.
+      In this case, ``:oversubscribe`` was effectively ignored.
+  - Various fixes to the ``openmpi.spec`` file to fix issues with rpm generation.
```
Comment (jsquyres) [inline docs/news/news-v5.0.x.rst:50]: You need another blank line before this one

File: docs/news/news-v5.0.x.rst [hunk 3]
```diff
@@ -40,24 +40,27 @@ Open MPI version 5.0.0rc5
...
-  - Fixed several memory leaks in the ``UCX`` component.
-  - Fixed a bug where ``autogen.pl --force`` would fail. 
-  - Fixed a large number of warnings when compiling on macOS.
-  - Fixed two dead links in ``HACKING.md``. Thanks to Lachlan Bell for finding and fixing.
-  - Updated various documentation to rename ``master`` to ``main`` where relevant.
+  - Fixed a bug in one-sided ``UCX`` calls where not all in-flight messages
+    would be flushed before cleanup.
+
+  - Build fixes - the following builds options with Open MPI were fixed:
+    - ``USNIC`` - (``configure --with-usnic=..``)
```
Comment (jsquyres) [inline docs/news/news-v5.0.x.rst:56]: You need a blank line before this one.  Also:

```suggestion
    - ``usNIC`` - (``configure --with-usnic=..``)
```

File: docs/news/news-v5.0.x.rst [hunk 4]
```diff
@@ -155,13 +160,14 @@ Changes since rc4:
...
     to this effort.
-  - Various ``README.md`` fixes - thanks to: Yixin Zhang, Samuel Cho,
-    Robert Langfield, Alex Ross, Sophia Fang, mitchelltopaloglu, Evstrife,
-    and Hao Tong for their contributions.
+  - Various ``README.md`` and ``HACKING.md`` fixes - thanks to: Yixin Zhang, Samuel Cho,
+    Robert Langfield, Alex Ross, Sophia Fang, mitchelltopaloglu, Evstrife, Hao Tong
+    and Lachlan Bell for their contributions.
   - Various CUDA documentation fixes. Thanks to Simon Byrne for finding
     and fixing these typos.
 - Build updates and fixes:
```
Comment (jsquyres) [inline docs/news/news-v5.0.x.rst:168]: Missing a blank line before this one, too.

**Review by jsquyres — CHANGES_REQUESTED** [review]

File: docs/news/news-v5.0.x.rst (hunk 4 above)
Comment (awlauria) [reply docs/news/news-v5.0.x.rst:168]: Ugh - forgot about that syntax. thanks.

**Review by jsquyres — APPROVED** [review]

**Pull request merged**